### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by hewrtarkhany**
Assignment1 URL : https://nice-river-0685dad0f.1.azurestaticapps.net
Partner : Ben Burbank

EXTRA CREDIT
3 Ideas
A. using a different language like Arabic or Persian  the cursor moves from right to left
B.  when the user login the Wordle app shows a demo of how to solve the problem
C. when the User gets the answer right there would a confetti or some celebratory theme shows 
F. some sort of counter for scoreboard or get points when you get the letter right

### CODE REVIEW

File: wordle-web/pages/index.vue [hunk 1]
```diff
@@ -12,7 +12,7 @@
       <v-card >  
 
         <v-card-title class="headline">
-          Welcome to the Game
+          Welcome to Hewr and Ben Game
```
Comment (BenagainB) [inline wordle-web/pages/index.vue:15]: These are the most epic names ever!

Comment (BenagainB) [inline wordle-web/pages/index.vue:15]: I looked at your web app deployment, and everything seems to be working well. I like the MDI-token you used in the  page menu header.

Comment (hewrtarkhany) [reply wordle-web/pages/index.vue:15]: thanks

Comment (hewrtarkhany) [reply wordle-web/pages/index.vue:15]: thanks you

**Comment by schuywalker** [comment]
I'll review!

**Comment by schuywalker** [comment]
Great job! Code is super clean - I honestly can't find anything to critique. All parts of the assignment are passed + extra credit. Localhost and the azure version of the website both work for me.

**Comment by mmwoodfo** [comment]
@BenagainB In the future, you and your partner only need one PR open. Make sure to close any PRs that you don't want to be graded. Typically the way I grade is the first PR I come across is the PR you get graded for.

**Comment by mmwoodfo** [comment]
Good job making an Assignment 1 branch. Great git branch management. Keep making separate branches for each assignment! Good job guys!

**Comment by mmwoodfo** [comment]
> Assignment1 URL : https://nice-river-0685dad0f.1.azurestaticapps.net Partner : Ben Burbank
> 
> EXTRA CREDIT 3 Ideas A. using a different language like Arabic or Persian the cursor moves from right to left B. when the user login the Wordle app shows a demo of how to solve the problem C. when the User gets the answer right there would a confetti or some celebratory theme shows F. some sort of counter for scoreboard or get points when you get the letter right

Nice ideas for the extra credit. If you guys have time, go ahead and log each of these ideas as issues under the issues tab: https://github.com/IntelliTect-Samples/EWU-CSCD379-2022-Spring/issues

File: wordle-web/pages/game.vue [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<template>
+    <v-row>
+        <v-col>
+            <v-card>
+
+            <h2>Game Same</h2>
+            <p>
+                this is a game that we play with words and ideas to find the right answer.
+               
+            </p>
```
Comment (mmwoodfo) [inline wordle-web/pages/game.vue:10]:
```suggestion

            <v-card-title class="text-h2">Game Same</v-card-title>
            <v-card-text>
                this is a game that we play with words and ideas to find the right answer.
            </v-card-text>
```

Be sure to take advantage of the Vuetify text and typography classes as well as the Vuetify components for text such as `v-card-text` or `v-card-title`

File: wordle-web/pages/game.vue [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+             <v-btn  color="red" nuxt to ="/"> GO BACK
```
Comment (mmwoodfo) [inline wordle-web/pages/game.vue:11]:
```suggestion
             <v-btn  color="error" nuxt to ="/"> GO BACK
```

Consider using your preset theme colors. I know we didn't talk about this until after Assignment 1 though, so good job with setting the colors!

**Pull request closed without merging**